app/models/poster.rb

```diff
@@ -1,3 +1,12 @@
 class Poster < ApplicationRecord
-
+  def self.sort_results(sort)
+    if sort === 'asc'
+      Poster.order(created_at: :asc)
+    elsif
+      sort === 'desc'
+      Poster.order(created_at: :desc)
+    else 
+      Poster.all
+    end
+  end
```

Comment:
Solid logic, good to put all the pieces together about being able to use ActiveRecord language inside of javascript.